ekncontent/ekncontent/eknc-media-object-model.c

```diff
@@ -89,6 +95,10 @@ eknc_media_object_model_set_property (GObject *object,
       priv->height = g_value_get_uint (value);
       break;
 
+    case PROP_PARENT_URI:
+      priv->parent_uri = g_value_dup_string (value);
```

Comment:
Not strictly necessary since it's construct-only, but it would be good practice to have `g_clear_pointer(&priv->parent_uri, g_free);` here.

Reply:
Good point, lets do that.